sites/partners/src/components/settings/SettingsViewHelpers.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { NextRouter } from "next/router"
+import { t } from "@bloom-housing/ui-components"
+import { Tabs } from "@bloom-housing/ui-seeds"
+
+export enum SettingsIndexEnum {
+  preferences = 0,
+  properties,
+}
+
+export const getSettingsTabs = (selectedIndex: SettingsIndexEnum, router: NextRouter) => {
+  const baseUrl = "/settings/"
+  return (
+    <Tabs
+      verticalSidebar
+      onSelect={(index) => void router.push(`${baseUrl}/${SettingsIndexEnum[index]}`)}
```

Comment:
@emilyjablonski just making a note here this touches on the conversation we had about the tabs accessibility and the Menu component that's still a todo for Seeds: https://github.com/bloom-housing/ui-seeds/issues/77